src/modules/Credits.ts

```diff
@@ -32,6 +31,16 @@ const Credits: Credit[] = [
         link: 'https://www.deviantart.com/torqupine',
         // NOTES: No notes about credit, have messaged asking for permission, awaiting confirmation
     },
+    {
+        resources: [
+            'Team Flare Grunt Duo',
+            'Brains & Brawn',
+            'Surfer',
+            'Golfer',
+        ],
+        name: 'IanWalder',
+        link: 'No link',
```

Comment:
Do you want to set this to your GitHub or something at least?
```suggestion
        link: 'https://github.com/IanWalder',
```

Reply:
```suggestion
        name: 'DaleArwin',
        link: 'https://www.deviantart.com/dalearwin',
```
I recently set up a DeviantArt account, so might as well plug. Thanks for the review